tables/Tables/TableCache.cc

```diff
@@ -47,19 +57,6 @@ PlainTable* TableCache::operator() (const String& tableName) const
     return 0;
 }
 
-PlainTable* TableCache::operator() (uInt index) const
-{
-    ScopedMutexLock sc(itsMutex);
-    return (PlainTable*) (tableMap_p.getVal (index));
-}
-
-uInt TableCache::ntable() const
-{
-    ScopedMutexLock sc(itsMutex);
-    return tableMap_p.ndefined();
-}
-
```

Comment:
removal of these two functions is not backward compatible and broke CASA compilation, I'll check if it is easy to fix within casa.